### PR TITLE
Fixes #178 - multi-line blocks in roxygen documentation should be ali…

### LINKED ISSUE #178: multi-line blocks in roxygen documentation should be aligned to the first non-tag word
currently, the R package has many `roxygen2` documentation blocks like this:

image

It would be better to standardize on aligning all lines to the first non-tag character, like this

image

To complete this issue, go through all of the files in `R-package/R/` and look for places where we have multi-line roxygen blocks. When changing the text, keep lines to under 100 characters in length.

### PR DESCRIPTION
**Pull request by chrsblck**
…gned to the first non-tag word

### CODE REVIEW

**Comment by chrsblck** [comment]
@jameslamb Should `\dontrun{` be moved to the line below?

https://github.com/uptake/uptasticsearch/blob/a3a50fc1578f46273828f5a78a8e9bfa106d42a2/r-pkg/R/get_fields.R#L17

File: r-pkg/R/unpack_nested_data.R [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
-#'   input data.table is unmodified.
+#'              This is a side-effect-free function: it returns a new data.table and the
+#'              input data.table is unmodified.
 #' @importFrom data.table as.data.table copy is.data.table rbindlist setnames
 #' @importFrom purrr map_if map_lgl map_int
 #' @export
 #' @param chomped_df a data.table
 #' @param col_to_unpack a character vector of length one: the column name to
-#'   unpack
+#'                      unpack
```
Comment (jameslamb) [inline r-pkg/R/unpack_nested_data.R]: for this one, could you move "unpack" up to the end of the preceding line? I think it was too aggressive for us to put it on its own line

**Comment by jameslamb** [comment]
> @jameslamb Should `\dontrun{` be moved to the line below?
> 
> https://github.com/uptake/uptasticsearch/blob/a3a50fc1578f46273828f5a78a8e9bfa106d42a2/r-pkg/R/get_fields.R#L17

yes please, good catch

**Comment by chrsblck** [comment]
@jameslamb Addressed both comments. I accidentally squashed the update too soon (bad habit). Sorry!

**Pull request merged**